src/Illuminate/Cache/ApcWrapper.php

```diff
@@ -18,6 +20,14 @@ class ApcWrapper
      */
     public function __construct()
     {
+        if (filter_var(ini_get('apc.enabled'), \FILTER_VALIDATE_BOOLEAN)) {
```

Comment:
`ini_get` might not be available.

Reply:
Could you elaborate? I am not able to find any specifics about when ini_get is not available.
Furthermore for example in 'ValidatePostSize' Middleware of laravel, ini_get is also used to determine the maximum post size, which would be a precedent? Thank you for your input.

Reply:
I'll not comment on that 'typo', serves me right for not addressing tests first :)

Reply:
> Could you elaborate? I am not able to find any specifics about when ini_get is not available.

The function could be disabled, which will raise an error.